### DISCUSSION TITLE
Add ``convert to issue`` button for checkboxes in other user's comment

### DISCUSSION THREAD

**Discussion by marwie** [community/community#15771] [Projects and Issues]
Hello,

when making checkbox list in an issue I can click the button to the right to convert the item to an issue. I like that feature as it allows to group and collect problems in one session and later decide which should be pulled out to be further discussed in a separate issue

![image](https://user-images.githubusercontent.com/5083203/165902143-ad5dad84-30d0-4df4-9a51-70454d9d038a.png)

Unfortunately that feature is missing when someone else adds comments with checkboxes. As a repository maintainer I'd still like to get the option to simple convert their points to issues like in the screenshot above

![image](https://user-images.githubusercontent.com/5083203/165902694-43f106a7-1cd1-4426-8ce6-482c4f556674.png)

**Comment by piotrczyz** [comment]
Definitely, it is the one I've used a lot in the traditional board and big downside of new solution for projects.

**Comment by oskar-rocksoft** [comment]
Would make our life much simpler, definitely worth implementing! Thanks :)
